### PR TITLE
fix (ui): Add `strategy` and `threshold` to the SQL test #11848

### LINKED ISSUE #11848: Add `strategy` and `threshold` to the SQL test
In https://github.com/open-metadata/OpenMetadata/pull/11847 we added 2 new `parameterDefinition` to the [tableCustomSQLQuery](https://github.com/open-metadata/OpenMetadata/blob/4af4f398db4faad73f6c3564c0dda65a313afc7b/openmetadata-service/src/main/resources/json/data/tests/tableCustomSQLQuery.json) test definition (1- `strategy` which is a list of 1 selectable option available in `optionValues`  and 2- `threshold`).

We need to reflect these changes from the UI end and display those 2 fields above the query editor

### PR DESCRIPTION
**Pull request by ShaileshParmar11**
### Describe your changes:

Fixes #11848 

Add strategy and threshold to the SQL test #11848

image
image

#
### Type of change:

- [ ] Bug fix
- [ ] Improvement
- [ ] New feature
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Documentation

#
### Checklist:

- [x] I have read the [**CONTRIBUTING**](https://docs.open-metadata.org/developers/contribute) document.
- [ ] My PR title is `Fixes <issue-number>: <short explanation>`
- [ ] I have commented on my code, particularly in hard-to-understand areas. 
- [ ] For JSON Schema changes: I updated the migration scripts or explained why it is not needed.

### CODE REVIEW

File: openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/components/ParameterForm.tsx [hunk 1]
```diff
@@ -37,135 +39,160 @@ const ParameterForm: React.FC<ParameterFormProps> = ({ definition, table }) => {
...
-          </Form.List>
-        );
+                      </Form.Item>
+                      <Button
+                        icon={
+                          <SVGIcons
+                            alt="delete"
+                            className="tw-w-4"
+                            icon={Icons.DELETE}
+                          />
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/components/ParameterForm.tsx:184]: Can we use AntD Icon component instead SVGIcons?

File: openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/components/ParameterForm.tsx [hunk 2]
```diff
@@ -37,135 +39,160 @@ const ParameterForm: React.FC<ParameterFormProps> = ({ definition, table }) => {
...
-                          className="tw-w-4"
-                          icon={Icons.DELETE}
+                    </span>
+                  }
+                  name={data.name}
+                  tooltip={data.description}>
+                  {fields.map(({ key, name, ...restField }) => (
+                    <div className="d-flex tw-gap-2 tw-w-full" key={key}>
+                      <Form.Item
+                        className="tw-w-11/12 tw-mb-4"
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/components/ParameterForm.tsx:161]: Can we avoid tailwind classes?

File: openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/components/ParameterForm.tsx [hunk 3]
```diff
@@ -37,135 +39,160 @@ const ParameterForm: React.FC<ParameterFormProps> = ({ definition, table }) => {
...
+            <Form.List
+              initialValue={[{ value: '' }]}
+              key={data.name}
+              name={data.name || ''}>
+              {(fields, { add, remove }) => (
+                <Form.Item
+                  key={data.name}
+                  label={
+                    <span>
+                      <span className="tw-mr-3">{data.displayName}:</span>
```
Comment (chirag-madlani) [inline openmetadata-ui/src/main/resources/ui/src/components/AddDataQualityTest/components/ParameterForm.tsx:147]: tailwind

**Comment by ShaileshParmar11** [comment]
Thanks for the feedback, I will take care of this in my next PR.!

**Review by chirag-madlani — APPROVED** [review]

**Pull request merged**